proposal/aiiuii/README.rst

```diff
@@ -0,0 +1,93 @@
+==============================
+Interbotix Ros Xsarms Toolkits
+==============================
+
+
+
+Basic Information
+=================
+
+The GitHub repository: https://github.com/aiiuii/Interbotix_Ros_Xsarms_Toolkits
+
+Problem to Solve
+================
+
+Recently, my supervising professor purchased a robotic arm for the lab. Unfortunately, no one in the lab had any experience with it, so learning how to use it ended up being one of my professor's assigned tasks.
```

Comment:
Have you got the permission to open source the code to control the robot?  Some vendors do not allow users to disclose their API.

NSD requires you to open source the code for the term project.  But if the robot vendor does not allow disclosure of their API, there will be a conflict of requirements.

Note, many vendors protect their control library / toolkit.  API is the interface to the toolkit, and may or may not have the same restriction.